spk/rutorrent/Makefile

```diff
@@ -27,7 +27,7 @@ DESCRIPTION = ruTorrent is a front-end for the popular Bittorrent client rTorren
 ADMIN_URL = /rutorrent/
 STARTABLE = yes
 DISPLAY_NAME = ruTorrent
-CHANGELOG = "<ol><li>Update to Python 3.10</li><li>Update to PHP 7.4</li><li>Remove tools from procps-ng</li><li>Fix support of DSM 7.</li><li>Fix torrent creation wizard (\#5288)</li><li>Add service port declarations</li></ol>"
+CHANGELOG = "<ol><li>Remove no outdated resources</li><li>Fix missing dependency declaration on Apache 2.4</li><li>Fixed invalid handling of volume customization upon installation</li><li>Fix handling of watch directory by creating it if it does not exist.</li></ol>"
```

Comment:
I don't believe DSM correctly parses `<ol>` and `<li>` tags. Perhaps just standard lines prefixed by numbers and a `<br/>` between then should be enough.

Reply:
Also shouldn't the first line be "Remove outdated resources"?

Reply:
Actually it works pretty fine the injecting of HTML from what I've been observing.

Fixing the first line !